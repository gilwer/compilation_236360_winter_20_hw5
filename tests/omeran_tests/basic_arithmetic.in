// basic arithmetic
void main() {
    int x = 1;
    printi(x); // 1
    int y = 1 + 2 + 3;
    printi(y); // 6
    int z = 1 + 2 * 3;
    printi(z); // 7
    int t = (1 + 2) * 3;
    printi(t); // 9
    int s = 1 + (2 * 3);
    printi(s); // 7
    int w = x + y - 2;
    printi(w); // 5
}