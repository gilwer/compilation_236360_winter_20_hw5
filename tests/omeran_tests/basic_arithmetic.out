1
6
7
9
7
5
